File: src/java1/q1.java
//Write a program to replace a substring inside a string with other string ?
package java1;

public class q1 {
    public static void main(String[] args) {
        String st = "hello";
        String sub="world";
        System.out.println(st.replace(st.substring(0,3),sub));


    }


}

File: Exercise1/q1.kt
package AssesmentOne

//Write a program to replace a substring inside a string with another string.
fun main() {
    println("Enter the String:")
    var str : String = readLine().toString()
    println("Enter indices for Substring:")
    println("Enter start Indices:")
    val start : Int = Integer.valueOf(readLine())

    println("Enter end Indices:")
    val end : Int = Integer.valueOf(readLine())

    println("Enter new String for replacement")
    val newStr: String = readLine().toString()

    val replacedString : String = str.replace(str.substring(start,end),newStr)
    println("String after Replacement")
    println(replacedString)

}